src/defrag.c

```diff
@@ -431,10 +431,10 @@ Defrag6Reassemble(ThreadVars *tv, DefragTracker *tracker, Packet *p)
     }
     PKT_SET_SRC(rp, PKT_SRC_DEFRAG);
 
-    int unfragmentable_len = 0;
-    int fragmentable_offset = 0;
-    int fragmentable_len = 0;
-    int ip_hdr_offset = 0;
+    uint16_t unfragmentable_len = 0;
```

Comment:
I'm a bit worried with these that we might overflow them. I don't see strict checks below. Did you review this?

Reply:
Will look into adding debug validations...